piptools/scripts/compile.py

```diff
@@ -291,6 +305,27 @@ def _determine_linesep(
     help="Specify a package to consider unsafe; may be used more than once. "
     f"Replaces default unsafe packages: {', '.join(sorted(UNSAFE_PACKAGES))}",
 )
+@click.option(
+    "--build-distribution",
+    "build_distributions",
+    multiple=True,
+    type=click.Choice(("editable", "sdist", "wheel")),
+    help="Name of a distribution to install build dependencies for; may be used more than once. "
```

Comment:
This also needs to be fixed: distribution is not equivalent to build deps. It's just an archive.

Reply:
I have updated the user facing terminology. Is it good now?